### PR TITLE
Moving Java-native-headless-v2 sample to this repo.

### PR DESCRIPTION
**Pull request by ramya25**
- Minor edits in the sample's README

### CODE REVIEW

File: active-directory-java-native-headless-v2/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Contributing to Azure samples
+
+Thank you for your interest in contributing to Azure samples!
+
+## Ways to contribute
+
+You can contribute to [Azure samples](https://azure.microsoft.com/documentation/samples/) in a few different ways:
+
+- Submit feedback on [this sample page](https://azure.microsoft.com/documentation/samples/active-directory-java-native-headless/) whether it was helpful or not.  
+- Submit issues through [issue tracker](https://github.com/Azure-Samples/active-directory-java-native-headless/issues) on GitHub. We are actively monitoring the issues and improving our samples.
```
Comment (navyasric) [inline active-directory-java-native-headless-v2/CONTRIBUTING.md]: These links would need to be updated to this repo

File: active-directory-java-native-headless-v2/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+### Step 2:  Clone or download this repository
+
+From your shell or command line:
+
+```shell
+`git clone https://github.com/Azure-Samples/ms-identity-java-desktop.git `
+```
+
+Go to `active-directory-java-native-headless-v2` folder
```
Comment (navyasric) [inline active-directory-java-native-headless-v2/README.md]: Can we rename this folder to "Call-MsGraph-WithUsernamePassword"

File: active-directory-java-native-headless-v2/pom.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <artifactId>public-client-msal4j-sample</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0</version>
+    <name>public-client-msal4j-sample</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>msal4j</artifactId>
+            <version>0.2.0-preview</version>
```
Comment (navyasric) [inline active-directory-java-native-headless-v2/pom.xml]: We should update to use the latest msal4j version. 0.7.0-preview.

**Comment by sangonzal** [comment]
@ramya25 @navyasric  Is there a reason we would like to move the sample to this repository? If we want to change the name, we can just update it in the other repo.

File: active-directory-java-native-headless-v2/README.md (hunk 2 above)
Comment (ramya25) [reply active-directory-java-native-headless-v2/README.md]: Sure. I will rename the folder

File: active-directory-java-native-headless-v2/pom.xml (hunk 3 above)
Comment (ramya25) [reply active-directory-java-native-headless-v2/pom.xml]: Sure

Comment (ramya25) [reply active-directory-java-native-headless-v2/pom.xml]: I think API changed in the new version. So we need to modify code accordingly.

File: active-directory-java-native-headless-v2/CONTRIBUTING.md (hunk 1 above)
Comment (ramya25) [reply active-directory-java-native-headless-v2/CONTRIBUTING.md]: updated

**Pull request merged**